test/parallel/test-http2-origin.js

```diff
@@ -70,6 +70,15 @@ const ca = readKey('fake-startcom-root-cert.pem', 'binary');
         }
       );
     });
+
```

Comment:
I would be nice to have a descriptive commnet about what this is testing.
Also 16383 is a magic number, maybe define MAX_HEADER_NUMBER,  and here use that + 1?